spec/configuration_spec.rb

```diff
@@ -47,70 +47,70 @@
       end
     end
 
-    describe "#minimum_coverage" do
+    shared_examples "checks coverage settings" do |coverage_setting|
       after :each do
         config.clear_coverage_criteria
       end
 
       it "does not warn you about your usage" do
         expect(config).not_to receive(:warn)
-        config.minimum_coverage(100.00)
+        config.send(coverage_setting, 100.00)
```

Comment:
hey can you please replace `send` with `public_send` basically everywhere here? :)

It's a pet peeve of mine, it helps us in only calling public methods so we won't accidentally make them private or something + highlighting we are indeed testing public methods and not private ones